enterprise/server/raft/leasekeeper/leasekeeper.go

```diff
@@ -342,7 +342,10 @@ func (lk *LeaseKeeper) RemoveRange(rd *rfpb.RangeDescriptor) {
 func (lk *LeaseKeeper) LeaseCount() int64 {
 	leaseCount := int64(0)
 	lk.leases.Range(func(key, value any) bool {
-		leaseCount += 1
+		la := value.(leaseAgent)
```

Comment:
fix a bug where we were counting all leases rather than just leases currently held